### PR TITLE
Update Reader.php

### PR DESCRIPTION
**Pull request by frontendschlampe**
fix errors in url generation

### CODE REVIEW

File: Module/Reader.php [hunk 1]
```diff
@@ -153,20 +153,16 @@ protected function compile()
...
 
 		if ($objNextArticle->numRows)
 		{
-            $arrNext = $objNextArticle->row();
-            $arrNext['parentJumpTo'] = $GLOBALS['objPage']->id;
-			$this->Template->nextArticle = array
-			(
-				'title' => $objNextArticle->title,
-				'href'	=> $this->Helper->generateUrl($arrNext),
-                'alias' => $objNextArticle->alias
```
Comment (psi-4ward) [inline Module/Reader.php]: We should not kill `alias` from the array

File: Module/Reader.php [hunk 2]
```diff
@@ -175,20 +171,16 @@ protected function compile()
...
 
 		if ($objPrevArticle->numRows)
 		{
-            $arrPrev = $objPrevArticle->row();
-            $arrPrev['parentJumpTo'] = $GLOBALS['objPage']->id;
-            $this->Template->prevArticle = array
-			(
-				'title' => $objPrevArticle->title,
-				'href'	=> $this->Helper->generateUrl($arrPrev),
-                'alias' => $objPrevArticle->alias
```
Comment (psi-4ward) [inline Module/Reader.php]: We should not kill `alias` from the array

Comment (backbone87) [inline Module/Reader.php]: it is still in there because the whole selected row is used as $arrPrev "base"

File: Module/Reader.php (hunk 1 above)
Comment (backbone87) [inline Module/Reader.php]: it is still in there because the whole selected row is used as $arrNext "base"

**Pull request closed without merging**